### PR TITLE
VRA-193 - remove covenant spells

### PR DESCRIPTION
**Pull request by nyy23**

### CODE REVIEW

File: SpellList.lua [hunk 1]
```diff
@@ -204,14 +185,8 @@ local spellListRetail = {
...
-		-- Covenant
-		[304971] = {type = 99}, --Divine Toll
-		[316958] = {type = 99}, --Ashen Hallow
-		[328204] = {type = 99}, --Vanquiser's Hammer
-		[328281] = {type = 99}, --Blessing of Winter
-		[328282] = {type = 99}, --Blessing of Spring
-		[328620] = {type = 99}, --Blessing of Summer
-		[328622] = {type = 99}, --Blessing of Autumn
+		[304971] = {type = 1}, --Divine Toll
+		[328620] = {type = 1}, --Blessing of Summer
```
Comment (Nitrak) [inline SpellList.lua:189]: This is not the ID for the talent. This is: 388007

File: SpellList.lua [hunk 2]
```diff
@@ -204,14 +185,8 @@ local spellListRetail = {
...
 		[343721] = {type = 1}, --Final Reckoning
-		-- Covenant
-		[304971] = {type = 99}, --Divine Toll
-		[316958] = {type = 99}, --Ashen Hallow
-		[328204] = {type = 99}, --Vanquiser's Hammer
-		[328281] = {type = 99}, --Blessing of Winter
-		[328282] = {type = 99}, --Blessing of Spring
-		[328620] = {type = 99}, --Blessing of Summer
-		[328622] = {type = 99}, --Blessing of Autumn
+		[304971] = {type = 1}, --Divine Toll
```
Comment (Nitrak) [inline SpellList.lua:188]: 375576

File: SpellList.lua [hunk 3]
```diff
@@ -172,11 +157,7 @@ local spellListRetail = {
 		[325197] = {type = 4}, --Invoke Chi-Ji, the Red Crane
 		[388615] = {type = 4}, --Restoral
 		[388686] = {type = 1}, --Summon White Tiger Statue
-		-- Covenant
-		[310454] = {type = 99}, --Weapons of Order
-		[325216] = {type = 99}, --Bonedust Brew
-		[326860] = {type = 99}, --Fallen Order
-		[327104] = {type = 99}, --Faeline Stomp
+		[327104] = {type = 1}, --Faeline Stomp
```
Comment (Nitrak) [inline SpellList.lua:160]: 388193

File: SpellList.lua [hunk 4]
```diff
@@ -58,11 +55,8 @@ local spellListRetail = {
 		[258925] = {type = 1}, --Fel Barrage
 		[263648] = {type = 2}, --Soul Barrier
 		[320341] = {type = 2}, --Bulk Extraction
-		-- Covenant
-		[306830] = {type = 99}, --Elysian Decree
-		[317009] = {type = 99}, --Sinful Brand
-		[370965] = {type = 99}, --The Hunt
-		[329554] = {type = 99}, --Fodder to the Flame
+		[306830] = {type = 1}, --Elysian Decree
```
Comment (Nitrak) [inline SpellList.lua:58]: 390163

File: SpellList.lua [hunk 5]
```diff
@@ -90,11 +84,9 @@ local spellListRetail = {
 		[203651] = {type = 3}, --Overgrowth
 		[205636] = {type = 1}, --Force of Nature
 		[319454] = {type = 1}, --Heart of the Wild
-		-- Covenant
-		[391528] = {type = 99}, --Convoke the Spirits
-		[323546] = {type = 99}, --Ravenous Frenzy
-		[325727] = {type = 99}, --Adaptive Swarm
-		[338142] = {type = 99}, --Lone Empowerment
+		[391528] = {type = 1}, --Convoke the Spirits
+		[325727] = {type = 1}, --Adaptive Swarm
```
Comment (Nitrak) [inline SpellList.lua:88]: 391888

File: SpellList.lua [hunk 6]
```diff
@@ -116,11 +108,7 @@ local spellListRetail = {
 		[264735] = {type = 2}, --Survival of the Fittest
 		[266779] = {type = 1}, --Coordinated Assault
 		[288613] = {type = 1}, --Trueshot
-		--Covenant
-		[308491] = {type = 99}, --Resonating Arrow
-		[324149] = {type = 99}, --Flayed Shot
-		[325028] = {type = 99}, --Death Chakram
-		[328231] = {type = 99}, --Wild Spirits
+		[325028] = {type = 1}, --Death Chakram
```
Comment (Nitrak) [inline SpellList.lua:111]: 375891

File: SpellList.lua [hunk 7]
```diff
@@ -142,11 +130,8 @@ local spellListRetail = {
 		[235450] = {type = 5}, --Prismatic Barrier
 		[342245] = {type = 2}, --Alter Time
 		[383121] = {type = 8}, --Mass Polymorph
-		-- Covenant
-		[307443] = {type = 99}, --Radiant Spark
-		[324220] = {type = 99}, --Deathborne
-		[314791] = {type = 99}, --Shifting Power
-		[314793] = {type = 99}, --Mirrors of Torment
+		[307443] = {type = 1}, --Radiant Spark
```
Comment (Nitrak) [inline SpellList.lua:133]: 376103

File: SpellList.lua [hunk 8]
```diff
@@ -142,11 +130,8 @@ local spellListRetail = {
...
+		[314791] = {type = 1}, --Shifting Power
```
Comment (Nitrak) [inline SpellList.lua:134]: 382440

File: SpellList.lua [hunk 9]
```diff
@@ -246,11 +221,7 @@ local spellListRetail = {
 		[316262] = {type = 1}, --Toughsteal
 		[328530] = {type = 1}, --Divine Ascension
 		[372835] = {type = 4}, --Lightwell
-		-- Covenant
-		[323673] = {type = 99}, --Mindgames
-		[324724] = {type = 99}, --Unholy Nova
-		[325013] = {type = 99}, --Boon of Ascendance
-		[327661] = {type = 99}, --Fae Guardian
+		[323673] = {type = 1}, --Mindgames
```
Comment (Nitrak) [inline SpellList.lua:224]: 375901

File: SpellList.lua [hunk 10]
```diff
@@ -270,11 +241,10 @@ local spellListRetail = {
 		[199754] = {type = 2}, --Riposte
 		[212182] = {type = 5}, --Smoke Bombe
 		[343142] = {type = 1}, --Dreadblades
-		-- Covenant
-		[323547] = {type = 99}, --Echoing Reprimand
-		[323654] = {type = 99}, --Flagellation
-		[328305] = {type = 99}, --Sepsis
-		[328547] = {type = 99}, --Serrated Bone Spike
+		[323547] = {type = 1}, --Echoing Reprimand
```
Comment (Nitrak) [inline SpellList.lua:244]: 385616

File: SpellList.lua [hunk 11]
```diff
@@ -270,11 +241,10 @@ local spellListRetail = {
...
+		[323654] = {type = 1}, --Flagellation
```
Comment (Nitrak) [inline SpellList.lua:245]: 384631

File: SpellList.lua [hunk 12]
```diff
@@ -270,11 +241,10 @@ local spellListRetail = {
...
+		[328305] = {type = 1}, --Sepsis
```
Comment (Nitrak) [inline SpellList.lua:246]: 385408

File: SpellList.lua [hunk 13]
```diff
@@ -270,11 +241,10 @@ local spellListRetail = {
...
+		[328547] = {type = 1}, --Serrated Bone Spike
```
Comment (Nitrak) [inline SpellList.lua:247]: 385424

File: SpellList.lua [hunk 14]
```diff
@@ -302,11 +272,8 @@ local spellListRetail = {
 		[198838] = {type = 3}, --Earthen Wall Totem
 		[207399] = {type = 4}, --Ancestral Protection Totem
 		[355580] = {type = 5}, --Static Field Totem
-		-- Covenant
-		[320674] = {type = 99}, --Chain Harvest
-		[324386] = {type = 99}, --Vesper Totem
-		[326059] = {type = 99}, --Primordial Wave
-		[328923] = {type = 99}, --Fae Transfusion
+		[326059] = {type = 1}, --Primordial Wave
```
Comment (Nitrak) [inline SpellList.lua:275]: 375982

File: SpellList.lua [hunk 15]
```diff
@@ -326,11 +293,7 @@ local spellListRetail = {
 		[267171] = {type = 1}, --Demonic Strength
 		[267217] = {type = 1}, --Nether Portal
 		[333889] = {type = 1}, --Fel Domination
-		-- Covenant
-		[312321] = {type = 99}, --Scouring Tithe
-		[321792] = {type = 99}, --Impending Catastrophe
-		[325289] = {type = 99}, --Decimating Bolt
-		[325640] = {type = 99}, --Soul Rot
+		[325640] = {type = 1}, --Soul Rot
```
Comment (Nitrak) [inline SpellList.lua:296]: 386997

File: SpellList.lua [hunk 16]
```diff
@@ -302,11 +272,8 @@ local spellListRetail = {
 		[198838] = {type = 3}, --Earthen Wall Totem
 		[207399] = {type = 4}, --Ancestral Protection Totem
 		[355580] = {type = 5}, --Static Field Totem
-		-- Covenant
-		[320674] = {type = 99}, --Chain Harvest
-		[324386] = {type = 99}, --Vesper Totem
-		[326059] = {type = 99}, --Primordial Wave
-		[328923] = {type = 99}, --Fae Transfusion
+		[326059] = {type = 1}, --Primordial Wave
+
```
Comment (Nitrak) [inline SpellList.lua:276]: remove newline

File: SpellList.lua [hunk 17]
```diff
@@ -90,11 +84,9 @@ local spellListRetail = {
...
 		[205636] = {type = 1}, --Force of Nature
 		[319454] = {type = 1}, --Heart of the Wild
-		-- Covenant
-		[391528] = {type = 99}, --Convoke the Spirits
-		[323546] = {type = 99}, --Ravenous Frenzy
-		[325727] = {type = 99}, --Adaptive Swarm
-		[338142] = {type = 99}, --Lone Empowerment
+		[391528] = {type = 1}, --Convoke the Spirits
+		[325727] = {type = 1}, --Adaptive Swarm
+
```
Comment (Nitrak) [inline SpellList.lua:89]: remove newline

**Review by Nitrak — CHANGES_REQUESTED** [review]
Added updated ID's for the covenant spells. The new talents use different IDs.

**Comment by nyy23** [comment]
Spell Correction Entries are taken from OmniCD

**Review by Nitrak — APPROVED** [review]

**Pull request merged**